pyproject.toml

```diff
@@ -1,46 +1,49 @@
-[tool.poetry]
+[project]
 name = "pypika-tortoise"
 version = "0.5.0"
 description = "Forked from pypika and streamline just for tortoise-orm"
-authors = ["long2ice <long2ice@gmail.com>"]
-license = "Apache-2.0"
-readme = "README.md"
+authors = [{name="long2ice", email="long2ice@gmail.com"}]
+license = {text="Apache-2.0"}
+keywords = ["postgresql", "Tortoise-ORM", "mysql", "sqlite", "sql"]
+include = ["CHANGELOG.md", "LICENSE", "README.md"]
+requires-python = ">=3.9"
+
+[project.urls]
 homepage = "https://github.com/tortoise/pypika-tortoise"
 repository = "https://github.com/tortoise/pypika-tortoise.git"
 documentation = "https://github.com/tortoise/pypika-tortoise"
-keywords = ["postgresql", "Tortoise-ORM", "mysql", "sqlite", "sql"]
+
+[tool.poetry]
+requires-poetry = ">=2.0"
```

Comment:
Is it going to prevent people with poetry < 2.0 from installing the package? If so, I don't think we want to do it.

Reply:
`requires-poetry = ">=2.0"` is a constraint for developing,
people use poetry v1 can add this package, even it's built by poetry v2.
Verified by the following commands:
```bash
git clone https://github.com/waketzheng/pypika-tortoise
cd pypika-tortoise
git checkout fix-poetry-v2
cd ..
pipx install --suffix=_v1 'poetry<2'
poetry_v1 new myproject
cd myproject
poetry_v1 add ../pypika-tortoise
```

Reply:
Separated the project section from this PR.